src/java/org/apache/cassandra/service/consensus/migration/TransactionalMigrationFromMode.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.service.consensus.migration;
+
+import org.apache.cassandra.service.consensus.TransactionalMode;
+
+public enum TransactionalMigrationFromMode
+{
+    none(null),
```

Comment:
```suggestion
    // No migration is in progress. The currently active transaction system could be either Accord or Paxos.
    none(null),
```

Reply:
Is this comment even technically true? When we finish a migration do we ever actually alter the schema to set this to none or are we basically "always" migrating with the ranges completely migrated and we only ever are in this state when do the very first migration?